.github/workflows/daily-version-check.yml

```diff
@@ -21,12 +21,20 @@ jobs:
       # the checks for other branches will still run to completion.
       fail-fast: false
       matrix:
-        target_branch:
-          # The list needs to be adjusted based on the branch where the workflow runs
-          - 'release-v2.8'
-          - 'release-v2.9'
-          - 'release-v2.10'
-          - 'release-v2.11'
+        include:
```

Comment:
Is
> We want to set up a workflow that runs periodically, ideally every night, to check for missing versions in the release branches:
>
>    The workflow in the release/v2.10 branch should verify that all versions from release/v2.9 and release/v2.8 exist in the current branch.
>    The workflow in the release/v2.9 branch should verify that all versions from release/v2.8 exist in the current branch.

meant to be included in this PR, or is this precursor work to enable that?

Reply:
That info is outdated. I just updated the description in the issue with the following: 

GitHub has the limitation that only scheduled jobs in the default branch are triggered automatically. To address this, the new solution is to run the workflow from the default branch, where it will fetch the data.json file from each pair of dev and release branches. The workflow will then compare and validate that all versions in the released data.json file exist in the corresponding dev file.

Reply:
PR description was updated with more information